dashboard/app/serializers/api/v1/pd/workshop_serializer.rb

```diff
@@ -44,6 +44,9 @@ def sessions
 
   def organizer
     {id: object.organizer.id, name: object.organizer.name, email: object.organizer.email}
+  rescue StandardError
```

Comment:
```suggestion
  rescue
```

It turns out rescuing `StardardError` [is the default behavior for `rescue`](https://thoughtbot.com/blog/rescue-standarderror-not-exception), so no need to specify it here.